### DISCUSSION TITLE
Dashboard Top upstreams not reflecting  new listing

### DISCUSSION THREAD

**Discussion by smurfsurfing** [AdguardTeam/AdGuardHome#7538] [Q&A] [answered]
I have updated upstream DNS servers listing on /mnt/dietpi_userdata/adguardhome/dietpi-unbound.conf and a few that were delete still showing up on the Dashboard top upstream.  I've rebooted multiple times, temporary disable blocking and re-enable, clear cache.  Anyone experience similar or a solution?

Thanks

**Comment by Dynamic5912** [comment, marked as answer]
> I have updated upstream DNS servers listing on /mnt/dietpi_userdata/adguardhome/dietpi-unbound.conf and a few that were delete still showing up on the Dashboard top upstream.  I've rebooted multiple times, temporary disable blocking and re-enable, clear cache.  Anyone experience similar or a solution?
> 
> 
> 
> Thanks

They will show there until you reset the stats for AGH.

**Comment by smurfsurfing** [comment]
Thank you
